README.md

```diff
@@ -1,2 +1,465 @@
 # terraform-docs-common
 Content for Terraform's documentation.
+
+- https://terraform.io/cloud-docs
+- https://terraform.io/plugin
+- https://terraform.io/registry
+
+## Running the Site Locally
```

Comment:
Could we please also have the following (similar to what we have on terraform-website?) 
- Table of Contents
- Contributions Welcome!
- Where the Docs Live

Reply:
that's fair and I support!